cmd/kube-apiserver/app/server.go

```diff
@@ -549,8 +548,7 @@ func BuildStorageFactory(s *options.ServerRunOptions) (*serverstorage.DefaultSto
 		s.Etcd.StorageConfig, s.Etcd.DefaultStorageMediaType, api.Codecs,
 		serverstorage.NewDefaultResourceEncodingConfig(api.Registry), storageGroupsToEncodingVersion,
 		// FIXME: this GroupVersionResource override should be configurable
-		// TODO we need to update this to batch/v1beta1 when it's enabled by default
-		[]schema.GroupVersionResource{batch.Resource("cronjobs").WithVersion("v2alpha1")},
+		[]schema.GroupVersionResource{batch.Resource("cronjobs").WithVersion("v1beta1")},
```

Comment:
What does this line do?

Reply:
This makes sure CronJob is stored with batch/v1beta1. If this is not present current mechanics will try to find codec for CronJob in batch/v1, which will fail. This needs to be fixed, I've spoked with @deads2k about it earlier today, but that won't happen in 1.8 timeframe, I guess.

Reply:
Does this change default storage version from batch/v2alpha1 to batch/v1beta1?